packages/vscode-css-languageservice/src/parser/cssParser.ts

```diff
@@ -359,14 +390,21 @@ export class Parser {
 	}
 
 	public _tryParseRuleset(isNested: boolean): nodes.RuleSet | null {
-		const mark = this.mark();
+		let mark = this.mark();
```

Comment:
```suggestion
		const mark = this.mark();
```

Doesn't look like this is reassigned